examples/with-google-analytics/components/Page.js

```diff
@@ -4,9 +4,9 @@ import Header from './Header'
 
 import * as gtag from '../lib/gtag'
 
-Router.onRouteChangeComplete = url => {
-  gtag.pageview(url)
-}
+Router.ready(() => {
+  Router.router.events.on('routeChangeComplete', url => gtag.pageview(url))
```

Comment:
Shouldn't this be `Router.events.on` (based on your change on canary)

Reply:
Basically I don't want to promote the use of the internal property if we don't have to 👍

Reply:
I think having `events` on the static router would be great idea, but my change didn't do that. It's only attached to the public router instance attached to the child context at this time. Will see about adding that.

Reply:
You're right. I guess the sample isn't installing the canary version, which is throwing me off.